moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -145,7 +145,8 @@ const NewProjectView = () => {
               {
                 phase_id: 1,
                 is_current_phase: true,
-                phase_start: format(Date.now(), "yyyy-MM-dd"),
```

Comment:
One last thing - there is compile warning about `format` no longer being used in this file.